blogposts/2021/optimizing-a-code-intelligence-commit-graph-2.md

```diff
@@ -2,20 +2,22 @@
 title: "Optimizing a code intelligence commit graph (Part 2)"
 author: Eric Fritz
 authorUrl: https://eric-fritz.com
+description: "We enabled Sourcegraph to resolve code intelligence queries on commits missing a code intelligence index, but ran into scalability challenges for users with large commit graphs. Here's how we unearthed and resolved the problem."
```

Comment:
@efritz I've taken a crack at the description but you will know better than me :)